### PR TITLE
reflect changes in security to optionals v4.12.0+

### PR DESCRIPTION
**Pull request by RobinKamps**
Just fixed this in my own project... - Resolves Issue #79

### CODE REVIEW

**Comment by poolebu** [comment]
I'm new to continuous integration, but wanting to learn. I'm wondering why did the travis checks failed ? I understand that 3 environments failed, how do you know where did they failed ? I don't see errors on the logs.

**Comment by RobinKamps** [comment]
It is just a lint error: 
https://travis-ci.org/hipster-labs/generator-jhipster-entity-audit/jobs/325892474#L7064

The ng-prefix is actually set the right way so i guess it is just a lint / travis issue -
perhaps a version error as stated in the log or it could be solved by replacing npm run lint with  esw webpack.config.* src tools; exit 0 etc..

File: travis/samples/gradle-ng1-mysql/.yo-rc.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
       "gatling",
       "protractor"
     ],
-    "jhiPrefix": "custom",
```
Comment (deepu105) [inline travis/samples/gradle-ng1-mysql/.yo-rc.json]: why is this changed? it should work with custom as well

**Comment by RobinKamps** [comment]
Update:
The ng-prefix was set as requested in the ng-style guide with a prefix "jhi" in this case.
But the prefix was configured as "custom" in travis/samples/gradle-ng1-mysql/.yo-rc.json.
So the last commit should fix two out of the three CI-Fails.

Can´t find the exact cause of the third failing test.
It is just a trailing whitespace error but on my system there are no trailing whitespaces in the whole yeoman template file at all...
ERROR: /home/travis/app/src/main/webapp/app/admin/entity-audit/entity-audit.component.ts[64, 1]: trailing whitespace

File: travis/samples/gradle-ng1-mysql/.yo-rc.json (hunk 1 above)
Comment (deepu105) [reply travis/samples/gradle-ng1-mysql/.yo-rc.json]: The correct fix is to use the jhiPrefix variable instead of hardcoded `jhi` prefix so that it will use what is set properly

Comment (RobinKamps) [reply travis/samples/gradle-ng1-mysql/.yo-rc.json]: sorry - my fault - i will revert this. Thank you for your review.

Comment (RobinKamps) [reply travis/samples/gradle-ng1-mysql/.yo-rc.json]: I reverted this back to "custom" - didn´t see your last advise - can you add the variable - or give me a hint about the syntax to access the variable ?
Thanks!

**Comment by RobinKamps** [comment]
Im stuck - please help.

**Comment by deepu105** [comment]
sure i'll push the required change


Thanks & Regards,
Deepu
